modules/apps/transfer/ibc_module.go

```diff
@@ -89,16 +90,21 @@ func (im IBCModule) OnChanOpenInit(
 		version = types.Version
 	}
 
-	if version != types.Version {
-		return "", errorsmod.Wrapf(types.ErrInvalidVersion, "expected %s, got %s", types.Version, version)
+	if !slices.Contains(types.SupportedVersions, version) {
+		return "", errorsmod.Wrapf(types.ErrInvalidVersion, "expected one of [%s, %s], got %s", types.Version1, types.Version, version)
 	}
 
 	// Claim channel capability passed back by IBC module
 	if err := im.keeper.ClaimCapability(ctx, chanCap, host.ChannelCapabilityPath(portID, channelID)); err != nil {
 		return "", err
 	}
 
-	return version, nil
+	if version == types.Version1 {
+		return types.Version1, nil
+	}
+
+	// default to latest supported version
+	return types.Version, nil
```

Comment:
Can we just return the `version` directly, not sure if I'm missing something but it reads a little confusingly to me. We default to ics20-2 on L90 if empty string is provided. 

```suggestion
	return version, nil
```